resources/decision_tree_GRCh38.json

```diff
@@ -397,6 +407,100 @@
       "outcomeFalse": {
         "nextNode": "exit_lb"
       },
+      "outcomeMissing": {
+        "nextNode": "non_coding"
+      }
+    },
+    "non_coding": {
```

Comment:
would it be better to start with GREEN-DB node since it is an aggregate method that used ReMM, ncER and FATHMM_MKL_NC? the GREEN-DB node could then forward to the other-non-coding-tools-node in case the GREEN-DB score is missing.

Reply:
from the green db docs:
Level Description
1 Rare variant (population AF < 1%) overlapping one of GREEN-DB regions
2 Level 1 criteria and overlap at least one functional element among transcription factors binding sites (TFBS),
DNase peaks, ultra conserved elements (UCNE)
3 Level 2 criteria and prediction score value above the suggested FDR50 threshold for at least one among
ncER, FATHMM MKL, ReMM
4 Level 3 critera and region constraint value greater or equal 0.7

I used this as inspiration, so the "non_coding" node is the level2 criteria, and the level 3 is represented by the green db node.